maas/resource_maas_block_device.go

```diff
@@ -386,7 +386,7 @@ func updateBlockDevicePartitions(client *client.Client, d *schema.ResourceData,
 				return err
 			}
 		}
-		if fsType := partition["fs_type"].(string); fsType != "" {
+		if fsType := partition["fs_type"].(string); fsType != "" && fsType != "raid" {
```

Comment:
this would be preferable in a separate PR, and explaining why this is needed?

Reply:
otherwise its try to partitioning my raid block device with "raid" fstype and API request will fail. I will reproduce the error message and paste it here.

Reply:
It would still be preferable to have this as a separate PR, land that one first, and rebase this.